src/helpers/linkQueries.ts

```diff
@@ -0,0 +1,110 @@
+import Fuse from "fuse.js";
+import { Pool } from 'pg';
+import { Config } from "../config";
+import { logger } from "../logger";
+
+export interface Link {
+  id: number,
+  shorten_link: string,
+  url: string,
+}
+
+var localLinks: Link[], pool: Pool;
+
+const ConnectToDB = async () => {
+  try {
+    pool = new Pool({
+      host: Config?.db_host,
+      user: Config?.db_user,
+      database: Config?.db_name,
+      password: Config?.db_password,
+      port: Config?.db_port
+    });
+    await pool.connect();
+    await SeedingData()
+  } catch (err) {
+    logger.debug(err);
+  }
+};
+
+const SeedingData = async () => {
+  try {
+    await pool.query('CREATE TABLE IF NOT EXISTS links (id bigserial primary key, shorten_link text, url text);')
+  } catch (err) {
+    logger.debug({ err })
```

Comment:
This should be a fatal error. Database creation is a required feature for this IMO